features/fixtures/mazerunner/src/main/cpp/bugsnags.cpp

```diff
@@ -396,4 +396,28 @@ Java_com_bugsnag_android_mazerunner_scenarios_CXXGetJavaDataScenario_activate(JN
     __builtin_trap();
 }
 
+JNIEXPORT void JNICALL
+Java_com_bugsnag_android_mazerunner_scenarios_MultiProcessHandledCXXErrorScenario_activate(JNIEnv *env,
+                                                                                           jobject instance) {
+  bugsnag_notify_env(env, (char *)"Achtung!",
```

Comment:
Can we change this to something more meaningful, so if we get payloads spilling or duplicating into other tests we can tell where they originated?

Reply:
```suggestion
  bugsnag_notify_env(env, (char *)"activate",
```